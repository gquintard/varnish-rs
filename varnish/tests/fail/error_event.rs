#[varnish::vmod]
mod event_dup {
    #[event]
    pub fn event_fn1() {}

    #[event]
    pub fn event_fn2() {}
}

fn main() {}
