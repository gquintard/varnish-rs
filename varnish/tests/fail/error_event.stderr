error: Only one event handler is allowed
 --> tests/fail/error_event.rs:7:12
  |
7 |     pub fn event_fn2() {}
  |            ^^^^^^^^^
